### PR TITLE
Update maths/Volume.java

### PR DESCRIPTION
**Pull request by Harpia-Vieillot**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Review by siriak — CHANGES_REQUESTED** [review]
Looks good. Please fix the build check.

File: src/test/java/com/thealgorithms/maths/VolumeTest.java [hunk 1]
```diff
@@ -31,5 +31,8 @@ public void volume() {
 
         /* test pyramid */
         assertTrue(Volume.volumePyramid(10, 3) == 10.0);
+        
+        /* test ellipsoid */
+        assertTrue(volumeEllipsoid(2, 5, 7) == 11.666666666666666);
```
Comment (debasishbsws) [inline src/test/java/com/thealgorithms/maths/VolumeTest.java:36]:
```suggestion
        assertTrue(Volume.volumeEllipsoid(2, 5, 7) == 11.666666666666666);
```

Comment (debasishbsws) [inline src/test/java/com/thealgorithms/maths/VolumeTest.java:36]:
```suggestion
        assertTrue(Volume.volumeEllipsoid(2, 5, 7) == 11.666666666666666);
```

**Review by debasishbsws — CHANGES_REQUESTED** [review]
Ellipsoid Volume formula is wrong
https://www.omnicalculator.com/math/ellipsoid-volume#:~:text=Volume%20%3D%204/3%20*%20%CF%80%20*%20A%20*%20B%20*%20C%2C
  
Check this, change the code, and test accordingly.

File: src/test/java/com/thealgorithms/maths/VolumeTest.java (hunk 1 above)
Comment (Harpia-Vieillot) [reply src/test/java/com/thealgorithms/maths/VolumeTest.java]: > This should fix the build fail

It isn't working bro...

Comment (Harpia-Vieillot) [reply src/test/java/com/thealgorithms/maths/VolumeTest.java]: > This should fix the build fail

It isn't working.

**Comment by Harpia-Vieillot** [comment]
> Looks good. Please fix the build check.

I am unable to find where the build check is failing.

**Comment by siriak** [comment]
It says
Error:  COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
Error:  /home/runner/work/Java/Java/src/main/java/com/thealgorithms/maths/Volume.java:[98,63] illegal start of type
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  7.021 s
[INFO] Finished at: 2022-11-20T14:33:13Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project Java: Compilation failure
Error:  /home/runner/work/Java/Java/src/main/java/com/thealgorithms/maths/Volume.java:[98,63] illegal start of type
Error:  -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Error: Process completed with exit code 1.

**Pull request closed without merging**